docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM openjdk:8u181-jdk-stretch
+
+LABEL MAINTAINER=fx19880617@gmail.com
+
+ARG PINOT_BRANCH=master
+ARG PINOT_GIT_URL="https://github.com/apache/incubator-pinot.git"
+RUN echo "Trying to build Pinot from [ ${PINOT_GIT_URL} ] on branch [ ${PINOT_BRANCH} ]"
+ENV PINOT_HOME=/opt/pinot
```

Comment:
Just try to understand, can we pass these through environment variables?

Reply:
This is image path, it could be passed from outside, but not really necessary.